src/conversions.jl

```diff
@@ -777,6 +777,11 @@ function convert_attribute(cs::Union{Tuple, Pair}, ::key"colormap", n::Int=2)
     return to_colormap([to_color.(cs)...], n)
 end
 
+function convert_attribute(cs::Tuple{Union{Symbol, AbstractString}, Real}, ::key"colormap", n::Int=30)
+    return RGBAf0.(to_colormap(cs[1]), cs[2]) # We need to rework this to conform to the backend interface.
```

Comment:
># We need to rework this to conform to the backend interface.

What needs to be reworked here?

Btw, while we're at it, maybe we can add also one for n? Maybe a named tuple?

Reply:
So my primary concern with the vector of rgb solution we currently have, is that we also have the sampler interface, and PlotUtils also has their own (new) interface for scaling etc.

It just feels like a lot of duplicated code when we should standardize on one implementation.

Reply:
Yeah... I will need to clean this up soon ;) This is my second priority after merging the font layout!